subsys/bluetooth/common/Kconfig

```
@@ -6,7 +6,8 @@
 
 config BT_BUF_ACL_TX_SIZE
 	int "Maximum supported ACL size for outgoing data"
-	range 27 251
+	range 27 251 if BT_LL_SW_SPLIT
+	range 27 1300
```

Where does the 1300 come from? I assume that is to use the same value for TX and RX, but I can't find the value in the spec. 

As far as I can tell, the maximum size is simply 0xFFFF.

---

It's taken from the RX size yes. I thought it would be reasonable that both be the same.

---

I agree, although now I'm more curious as to why that was set to 1300 instead of the maximum supported value from the HCI spec :D @cvinayak do you know?

---

HCI Command and Event has a max length of 255 bytes, HCI Data length is 2 octets and in the range minimum of 27 bytes to maximum 65535 bytes.

Please update the Rx buffer range too.

```suggestion
	range 27 65535
```